e2e/tests/api-driven/src/flowStatusHistory/helpers.ts

```diff
@@ -13,8 +13,9 @@ export const setup = async () => {
 };
 
 export const cleanup = async () => {
-  await $admin.user._destroyAll();
+  await $admin.flow._destroyPublishedAll();
   await $admin.flow._destroyAll();
+  await $admin.user._destroyAll();
```

Comment:
Order of cleanup matters - users create published flows and published flows are links to flows, so if we try to delete them before their published flows are deleted, we'll throw an error - see planx-core PR description with example https://github.com/theopensystemslab/planx-core/pull/644